src/app/_components/KeyMemberCard.tsx

```diff
@@ -0,0 +1,48 @@
+import Image from 'next/image'
+
+import { LinkedinLogo } from '@phosphor-icons/react/dist/ssr'
+
+import { Heading } from '@/components/Heading'
+import { Icon } from '@/components/Icon'
+import { TextLink } from '@/components/TextLink'
+
+import type { MemberData } from '@/types/memberType'
+
+type KeyMemberCardProps = MemberData
+
+export function KeyMemberCard({
+  name,
+  title,
+  linkedin,
+  image,
+}: KeyMemberCardProps) {
+  return (
+    <li className="flex rounded-lg border border-brand-500 p-1">
+      <Image
+        src={image}
+        alt={`Photo of ${name}`}
+        sizes="10rem"
+        className="aspect-[3/4] w-32 rounded object-cover"
+      />
+
+      <div className="relative m-3 grow">
+        <Heading isClamped tag="h3" variant="lg">
+          {name}
+        </Heading>
+
+        <p className="mt-1 text-brand-300">{title}</p>
+
+        <TextLink
+          href={linkedin}
+          aria-label={`Visit ${name}'s LinkedIn profile.`}
+          className="absolute inset-0 rounded-lg focus:outline focus:outline-2 focus:outline-brand-100"
+        >
+          <span className="absolute bottom-0 left-0 flex items-center gap-2 text-brand-300">
```

Comment:
Perhaps we can align this implementation more closely to the way `GovernanceCalendarCards` is one with `Card.Link`?

Reply:
I like it! I wasn't aware of `Card.Link`.

I ended up moving the link up to make the whole card clickable.

Reply:
Maybe check ~

```
          <li
            key={id}
            className="flex flex-col rounded-lg border border-blue-500 p-1 sm:flex-row"
          >
            <Calendar startDate={start.dateTime} />
            <div className="relative">
              <div className="relative flex flex-1 flex-col gap-3 p-4">
                <div className="flex gap-2">
                  <Badge
                    borderColor="brand-100"
                    icon={Clock}
                  >{`UTC ${startTime} - ${endTime}`}</Badge>
                  <Badge>Zoom</Badge>
                </div>
                <Heading tag="h3" variant="lg">
                  {summary}
                </Heading>
              </div>
              <div className="mb-10 mt-3">
                <Card.Link
                  href={htmlLink}
                  icon={CalendarPlus}
                  text="Add to Google Calendar"
                />
              </div>
            </div>
          </li>
```

Reply:
I fixed it with this commit https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/152/commits/80e7b9ed1e015036518ec74d866e6f1018a8b44f

I didn't use `Card.Link` because it needs a refactor to work for this use case. We can do it later; it's out of scope for this PR.

As mentioned earlier today, `GovernanceCalendarCards` is not fully clickable. Once `Card.Link` is refactored, we can implement it properly there, too.

https://www.notion.so/filecoin/FF-V4-Final-checks-and-cleanups-e09f9b0526c646b7ba533215964ed198?pvs=4#2efb468ceef14f8dab23b84b2007cf65